File: php/examples/jacobsthal_numbers.php
<?php

require_once('../lib/sequences.php');

$i = 0;
foreach (ClounceMath\Sequences::jacobsthalNumberNonRecursiveGenerator(50) as $sequenceNumber) {
    printf("J(%2s): %-30s", $i++, $sequenceNumber);
    if ($i % 3 === 0) {
        echo PHP_EOL;
    }
}

File: csharp/examples/JacobsthalNumbers.cs
namespace ClounceMathExamples {
    using ClounceMath;
    using System;

    sealed class JacobsthalNumbers {
        public static void RunSequence() {
            int i = 0;
            foreach (ulong number in Sequences.JacobsthalNumberNonRecursiveGenerator(50)) {
                Console.Write("J({0,2}): {1,-30}", i, number);
                if (++i % 3 == 0) {
                    Console.WriteLine();
                }
            }
            Console.WriteLine();
        } 
    }
}